docs/METRICS.md

```diff
@@ -16,6 +16,8 @@ We provide an implementation of the Scale-Invariant Signal-to-Noise Ratio in PyT
 No specific requirement is needed for this metric. Please activate the metric at the
 evaluation stage with the appropriate flag:
 
+**Warning:** As we also allow SISNR to be used as a training loss, we actually report minus the SISNR, e.g. negative values are best.
+
```

Comment:
continued:
**Warning** Please be aware that in the case of using SISNR as a training loss, the reported values are represented as negative. This implies that the best results are indicated by lower (negative) values.

assuming they do not change the original intention or message.